debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/RowMapperTest.java

```diff
@@ -33,6 +33,9 @@
 
 @SkipWhenAdapterNameIsNot(value = AdapterName.LOGMINER)
 public class RowMapperTest {
+
+    private static final Scn SCN_ONE = new Scn(BigDecimal.ONE);
```

Comment:
I think there's a constant for this in main code already?

Reply:
I didn't see one specifically for the value of `1`, we do have one for the value of `0` however.